### DISCUSSION TITLE
Status of Get-PnPUPABulkImportStatus

### DISCUSSION THREAD

**Discussion by EA12** [pnp/powershell#1698] [General]
Please add all possible status of CmdLet Get-PnPUPABulkImportStatus to the documentation.

What i found so far through testing:

- Submitted
- Queued
- Succeeded

but what about failures?
Errored?
Failed?

**Comment by stevebeauge** [comment]
+1 for the suggestion

If it can help : https://learn.microsoft.com/en-us/previous-versions/office/sharepoint-csom/mt643017(v=office.15)

(a bit late)
